backend/s3/s3.go

```diff
@@ -66,7 +66,7 @@ type s3Blob struct {
 
 func (bb *s3Blob) Write(p []byte) (int, error) {
 	if bb.final {
-		return 0, errors.New("blob already closed")
+		return 0, errors.New("Blob already closed")
```

Comment:
Same as above: don't start error strings with a capital letter, and don't fix unneeded things in an unrelated commit ;)